tests/cli_test.py

```diff
@@ -253,6 +253,35 @@ def test_cli_execution_add_change(self) -> None:
         self.assertIn("0.1.0", result.output)
 
 
+class NextVersionCommandTest(CommandTest):
+
+    def test_cli_execution_next_version(self) -> None:
+        runner = CliRunner()
+        result = runner.invoke(cli=cli, args=["next-version"])
+        assert result.exit_code == -1
+        assert "Error: No changes to release. Skipping release process." in result.output
+
+    def test_cli_execution_next_change(self) -> None:
+        commands = [
+            ["add-change", "--type", "minor", "--description", "This is my minor description"],
+            ["next-version"]
+        ]
+
+        result = command_processor(commands, self.directory_name)
+        self.assertIn("0.1.0", result.output)
```

Comment:
I haven't tried it yet, but I suppose these tests work because it is detecting the substring `0.1.0`. I guess the text displayed in the command is `Releasing version: 0.0.0 -> 0.1.0`, rather than `0.1.0` which is what we want. Would you mind confirming?

Reply:
Agreed.